### PR TITLE
feat: Disclosure Component

### PR DESCRIPTION
**Pull request by pcberg**

### CODE REVIEW

File: src/components/PageContent/Disclosure/Disclosure.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    <div
+      id={id}
+      className={classNames(disclosureClass, className)}
+      ref={disclosureWrapDiv}
+      {...attrs}
+    >
+      <button
+        className="rvt-disclosure__toggle"
+        onClick={toggleDisclosure}
+        aria-expanded="false"
```
Comment (johglove) [inline src/components/PageContent/Disclosure/Disclosure.jsx:87]: Believe this needs to reflect the open state of the component.

**Review by johglove — CHANGES_REQUESTED** [review]
Currently the chevron icon added to the title does not reflect the open state of the disclosure.

File: src/components/PageContent/Disclosure/Disclosure.jsx (hunk 1 above)
Comment (pcberg) [reply src/components/PageContent/Disclosure/Disclosure.jsx]: @johglove thanks for catching that, I fixed it. Let me know what you think!

**Review by burnumd — APPROVED** [review]

**Review by johglove — APPROVED** [review]

**Pull request merged**